test/test.config.js

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+const {config} = require('bedrock');
+const path = require('path');
+const axios = require('axios');
+const nodeAdapter = require('axios/lib/adapters/http');
+axios.defaults.adapter = nodeAdapter;
+require('bedrock-payment-paypal');
```

Comment:
Please copy or move this line to ~L7 of `test.js`.  Without doing this, dependency issues related to bedrock-payment-paypal get caught up in error handling related to `test.config.js`.  It should be required in test.js so that those errors are surfaced properly and are not obfuscated by that error handling.

Reply:
Good catch I moved the require statement into test.js here: https://github.com/digitalbazaar/bedrock-payment-paypal/pull/5/commits/91df6c5cd94b14bbc054dcb1fc97adc7c22162f4